.ci/check-for-chart-changes.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# check for chart changes to speedup ci
+#
+
+set -o errexit
+set -o pipefail
+
+CHART_REPO="https://github.com/kiwigrid/helm-charts.git"
```

Comment:
That should be replaced with the `https://github.com/eclipse/packages.git`, correct?

Reply:
fixed